lib/stbc.gd

```diff
@@ -216,20 +216,32 @@ DeclareGlobalName( "DefaultStabChainOptions" );
 
 #############################################################################
 ##
-#F  StabChainBaseStrongGenerators( <base>, <sgs>, <one> )
+#F  StabChainBaseStrongGenerators( <base>, <sgs>[, <one>] )
 ##
 ##  <#GAPDoc Label="StabChainBaseStrongGenerators">
 ##  <ManSection>
-##  <Func Name="StabChainBaseStrongGenerators" Arg='base, sgs, one'/>
+##  <Func Name="StabChainBaseStrongGenerators" Arg='base, sgs[, one]'/>
 ##
 ##  <Description>
 ##  Let <A>base</A> be a base for a permutation group <M>G</M>, and let
 ##  <A>sgs</A> be a strong generating set for <M>G</M> with respect to
 ##  <A>base</A>; <A>one</A> must be the appropriate identity element of
-##  <M>G</M> (see <Ref Attr="One"/>, in most cases this will be <C>()</C>).
-##  This function constructs a stabilizer chain without the need to find
+##  <M>G</M> (see <Ref Attr="One"/>; in most cases this will be <C>()</C>).
+##  This function constructs a stabilizer chain corresponding to the given
+##  base and strong generating set without the need to find
 ##  Schreier generators;
 ##  so this is much faster than the other algorithms.
+##  <P/>
+##  If <A>sgs</A> is nonempty, then the third argument <A>one</A> is
+##  optional; if <A>one</A> is not given in this case, then the identity
+##  element of <M>G</M> is taken to be the <Ref Attr="One" Style="Text"/>
+##  of the first entry of <A>sgs</A>.
```

Comment:
I found the reference to `G` a bit confusing, I thought "wait, `G` is not an argument, is it?" (But that may just be me). Maybe instead do this?
```suggestion
##  optional; if <A>one</A> is not given in this case, then the
##  <Ref Attr="One" Style="Text"/> of the first entry of <A>sgs</A> is taken
## as the identity element.
```